### PR TITLE
Isolate what's breaking stuff.

### PR DESCRIPTION
**Pull request by JohnMcLear**
Find a better way to do this.

### CODE REVIEW

File: static/js/client.js [hunk 1]
```diff
@@ -1,10 +1,14 @@
 var root = '../..';
 
+/* the below 3 lines break stuff */
+/*
 $.runScript(root+'texlive/website/pdftex.js/release/pdftex-webworker.js');
```
Comment (manuels) [inline static/js/client.js:5]: Is jquery not available in etherpad-lite?

Comment (JohnMcLear) [inline static/js/client.js:5]: it is, but...

a) they were outside of the documentReady
b) I think the documentReady might fire before $ is available?   *Not sure..

Comment (manuels) [inline static/js/client.js:5]: Nope, I moved it into documentReady, same error

**Comment by manuels** [comment]
I just committed some code (proof of concept). On my machine it works fine now.
Tell me if you run into trouble.

**Pull request closed without merging**